veejay-current/veejay-server/doc/README.audio.md

```diff
@@ -0,0 +1,93 @@
+                                                             
+                                                             
+       ,---,                                                 
+      '  .' \                         ,---,  ,--,            
+     /  ;    '.             ,--,    ,---.'|,--.'|    ,---.   
+    :  :       \          ,'_ /|    |   | :|  |,    '   ,'\  
+    :  |   /\   \    .--. |  | :    |   | |`--'_   /   /   | 
+    |  :  ' ;.   : ,'_ /| :  . |  ,--.__| |,' ,'| .   ; ,. : 
+    |  |  ;/  \   \|  ' | |  . . /   ,'   |'  | | '   | |: : 
+    '  :  | \  \ ,'|  | ' |  | |.   '  /  ||  | : '   | .; : 
+    |  |  '  '--'  :  | : ;  ; |'   ; |:  |'  : |_|   :    | 
+    |  :  :        '  :  `--'   \   | '/  '|  | '.'\   \  /  
+    |  | ,'        :  ,      .-./   :    :|;  :    ;`----'   
+    `--''           `--`----'    \   \  /  |  ,   /          
+                                  `----'    ---`-'           
+                                                             
+
+
+AUDIO support
+-------------
+
+Veejay has limited audio support aka no fancy effects on sound. Despite that,
+veejay will keep in sync the audio channel's of your clip, and transport it trough
+[JACK - jackaudio.org](http://jackaudio.org/) a famous and robust low latency audio server.
+
+
+Prepare
+-------
+
+You need an AVI file with an audio track encoded in signed PCM WAVE, 44-48Khz, 2 channels (stereo) (16 bit)
```

Comment:
the audio data in veejay is unsigned, but it doesn't really matter. the format is PCM WAV indeed, 4 byte per sample (2 byte per channel, 2 channels)

Reply:
I found that ffmpeg do not allow me to use pcm_u16le with AVI
```
Could not write header for output file #0 (incorrect codec parameters ?): Operation not permitted
```

And digging the web :
> PCM unsigned 16bits data. AVI/mov don't support this (they support signed 16 bits PCM data, or 8 bits unsigned PCM data, but not unsigned 16-bits PCM data)
https://stackoverflow.com/a/31856297

I did't find more revelant source about signed/unsigned pcm in avi ...